kuka_kr16_support/urdf/kr16_2_macro.xacro

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:property name="DEG2RAD" value="0.017453292519943295"/>
+
+  <xacro:macro name="kuka_kr16_2" params="prefix">
+    <!-- LINKS -->
+    <!-- base link -->
+    <link name="${prefix}base_link">
+      <inertial>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <mass value="2"/>
+        <inertia ixx="0.01" ixy="0" ixz="0" iyy="0.01" iyz="0" izz="0.01" />
+      </inertial>
```

Comment:
These are values just to keep gazebo happy. I am not sure if we will be able to get the real values. 
Regarding your comment on #16 I will take a look at  ros-industrial/abb_experimental#18 and come with comments.